packages/component/src/Styles/CustomPropertyNames.ts

```diff
@@ -0,0 +1,16 @@
+const CustomPropertyNames = Object.freeze({
```

Comment:
Using `Object.freeze` and `as const` is redundant. `as const` recursively freezes an object while `Object.freeze` freezes the top level.

Reply:
You are right... we probably only need `as const` for arrays, say, `Object.freeze([123, 'abc'] as const)`.